insights/parsers/du_usage.py

```diff
@@ -0,0 +1,55 @@
+"""
+du - Command
+============
+
+Module for the processing of output from the ``du`` command.  The base class
+``DiskUsage`` provides all of the functionality for all classes.
+Data is avaliable as rows of the output contained
+in one ``Record`` object for each line of output.
+
+Sample input data for the ``du -sh`` command looks like::
+22G     /var/lib/mongodb
+
+"""
+
+from .. import Parser, parser, LegacyItemAccess
+
+
+def parse_du_lines(du_content):
+
+    du_output = {}
+    """Parse contents of each line in ``du`` output.
+    Parse each line of ``du`` output ensuring that wrapped lines are
+    reassembled prior to parsing, and that mount names containing spaces
+    are maintained.
+
+    Parameters:
+        du_content (list): Lines of du output to be parsed.
+
+    """
+    for line in du_content:
+        line_split = line.split()
+        du_output[line_split[1]] = line_split[0]
+
+    return du_output
+
+
+@parser("du_-h")
+class DiskUsage(LegacyItemAccess, Parser):
+    def __len__(self):
+        return len(self.data)
+
+    def parse_content(self, content):
```

Comment:
Please add proper documentation for this. Refer other plugins for the same.

Reply:
ok, i'll add documentation to it.

Reply:
FYI, publicly available documentation is available here: http://insights-core.readthedocs.io/en/1.x/

Reply:
@bfahr  I'll look into it.